js/src/lib/components/InferenceWidget/widgets/AudioClassificationWidget/AudioClassificationWidget.svelte

```diff
@@ -147,12 +160,12 @@
 	}
 
 	onMount(() => {
-		const [exampleTitle, src] = getDemoInputs(model, ["example_title", "src"]);
+		const [exampleTitle, src, output] = getDemoInputs(model, ["example_title", "src", "output"]);
```

Comment:
the `getDemoInputs` function is pretty unnecessary and un-typed, i'd really like to remove it

Reply:
Indeed. wdyt about this change?
change signature of 
```ts
function applyInputSample(sample: WidgetExample)
```
to
```ts
function applyInputSample(sample: WidgetExample, opts: {isPreview?: boolean, ranRundomItem?: boolean})
```

so that `applyInputSample` fully replaces both [previewInputSample](https://github.com/huggingface/hub-docs/blob/7cfc6cd525b1f726e7394a95947f96b42d1082b3/js/src/lib/components/InferenceWidget/widgets/AudioClassificationWidget/AudioClassificationWidget.svelte#L146) & [getDemoInputs](https://github.com/huggingface/hub-docs/blob/7cfc6cd525b1f726e7394a95947f96b42d1082b3/js/src/lib/components/InferenceWidget/widgets/AudioClassificationWidget/AudioClassificationWidget.svelte#L163) ?

Reply:
actually, to replace `getDemoInputs`, it will be more like:
```ts
const randomExample = getRandomExample(model.widgets)
applyInputSample(randomExample)
```

Reply:
WDYT of this slightly better version of random example selection in #1021 already ? cc @SBrandeis too

Reply:
handled in https://github.com/huggingface/hub-docs/pull/1023